.env.example

```
@@ -33,3 +33,11 @@ PUSHER_APP_ID=your_app_id
 PUSHER_KEY=your_key
 PUSHER_SECRET=your_secret
 PUSHER_CLUSTER=your_cluster
+
+# --- Security Configuration ---
+ALLOWED_ORIGINS=http://localhost:3000,https://yourapp.com
+RATE_LIMIT_POINTS=100
+RATE_LIMIT_DURATION=60
+BODY_LIMIT=1mb
+TRUST_PROXY=true
+CSP_DEFAULT_SRC='self'
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)